pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -128,8 +128,8 @@ class DialogFooter extends React.Component {
     }
 
     render() {
-        const cancel_text = ('cancel_button' in this.props && this.props.cancel_button.text) ? this.props.cancel_button.text : _("Cancel");
-        const cancel_variant = ('cancel_button' in this.props && this.props.cancel_button.variant) ? this.props.cancel_button.variant : "link";
+        const cancel_text = ('cancel_button' in this.props?.cancel_button.text) ? this.props.cancel_button.text : _("Cancel");
```

Comment:
While you are at it, let's do this all the way:

    const cancel_text = this.props?.cancel_button?.text ?? _("Cancel")

and same for the next line.

Reply:
Done